src/web/server.js

```diff
@@ -168,6 +169,10 @@ const configureSentry = function configureSentry() {
   })
 }
 
+function configureMetrics (instance) {
+  instance.use(metrics.initialise())
```

Comment:
This method name is still pending on the library repository and so might need reverting, we've discussed and think its unlikely to change and should be straight forward if we need to. We'll learn more from having it progress through the various environments.